client/coral-admin/webpack.config.js

```diff
@@ -14,6 +15,13 @@ module.exports = Object.assign({}, devConfig, {
     ]
   },
   plugins: [
-    autoprefixer, precss
+    autoprefixer,
+    precss,
+    new webpack.DefinePlugin({
+      'process.env': {
+        'NODE_ENV': JSON.stringify('production'),
+        'VERSION': JSON.stringify(require("./package.json").version)
```

Comment:
oh, you use that instead of doing 

```
`"${thevalue}"`
```

makes more sense not to involve a json parser for that IMO